### ISSUE TITLE
Corridor routing issues near large parks (Mt. Pleasant, High Park)

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#755]
**What Happened**
From a user-submitted bug report:

> There is some link on Yonge Street that is not connected

This causes corridor routing near Mt. Pleasant on Yonge to fail.  A similar issue was observed when trying to reproduce near High Park on Bloor.

**What Should Happen**
These corridors should be routable.

**To Reproduce**
Steps to reproduce the bug:
1. Go to the map, search for "Yonge and St. Clair", and select the intersection
2. Click "Add Location", add "Yonge and Belsize"
3. Check "include intersections and midblocks between locations"
4. See error (red toast at bottom)

**Screenshots**
![Corridor routing error on Yonge near Mt. Pleasant](https://user-images.githubusercontent.com/151605/105370064-2d39fe80-5bd1-11eb-8fb9-14558820d485.png)

![Corridor mis-routed on Bloor near High Park](https://user-images.githubusercontent.com/151605/105370156-48a50980-5bd1-11eb-95b3-4f66cb6c1ed6.png)

**Additional Notes**
For the Bloor near High Park case, these are midblock IDs 30119699, 30119700.  A closer look at centreline geometry in the database reveals a strange quirk:

```sql
flashcrow=> select fnode, tnode from centreline.midblocks where "centrelineId" = 30119699;
  fnode   |  tnode
----------+----------
 13466332 | 30119698
(1 row)

flashcrow=> select fnode, tnode from centreline.midblocks where "centrelineId" = 30119700;
  fnode   |  tnode
----------+----------
 30119698 | 13466395
(1 row)

flashcrow=> select * from centreline.intersections where "centrelineId" = 30119698;
 centrelineId | centrelineType | classification | description | featureCode | geom | lat | lng
--------------+----------------+----------------+-------------+-------------+------+-----+-----
(0 rows)

flashcrow=> select * from centreline.midblocks where "centrelineId" = 30119698;
 aadt | centrelineId | centrelineType | featureCode | fnode | fromIntersectionName | geom | lat | lng | midblockName | roadId | tnode | toIntersectionName
------+--------------+----------------+-------------+-------+----------------------+------+-----+-----+--------------+--------+-------+--------------------
(0 rows)

flashcrow=> select * from gis.centreline_intersection where int_id = 30119698;
 gid | int_id | elev_id | intersec5 | classifi6 | classifi7 | num_elev | elevatio9 | elevatio10 | elev_level | elevation | elevatio13 | height_r14 | height_r15 | x | y | longitude | latitude | objectid | geom
-----+--------+---------+-----------+-----------+-----------+----------+-----------+------------+------------+-----------+------------+------------+------------+---+---+-----------+----------+----------+------
(0 rows)

flashcrow=> select * from gis.centreline where geo_id = 30119698;
 gid | geo_id | lfn_id | lf_name | address_l | address_r | oe_flag_l | oe_flag_r | lonuml | hinuml | lonumr | hinumr | fnode | tnode | fcode | fcode_desc | juris_code | objectid | geom
-----+--------+--------+---------+-----------+-----------+-----------+-----------+--------+--------+--------+--------+-------+-------+-------+------------+------------+----------+------
(0 rows)
```

In other words: both of these segments refer to a feature with ID `30119698`, but no such feature exists - this appears to be a pseudo-feature, included only so that these two segments have a common node.

**Comment by candu** [comment]
One possible fix: include these pseudo-features in the routing target, and let our existing `location !== null` check filter these out as well.

This could be achieved by dropping the `JOIN gis.centreline_intersection` clause from [this task](https://github.com/CityofToronto/bdit_move_etl/blob/master/scripts/airflow/tasks/centreline_conflation_target/A1_intersection_ids.sql), though we'd need to test to ensure this doesn't have unexpected consequences.

**Comment by candu** [comment]
We've found a way to make the fix described above work, but we're going to push this out to version 1.2 to focus on testing features for version 1.1.

**Comment by candu** [comment]
Double-checked after upgrading to TCL v2 - this issue is still happening.  (We didn't really expect that upgrade to fix this, but it's worth checking just in case there were changes to road network topology.)
